
This patch fixes a bug where the enable_acl=yes flag was not
being properly received by the File daemon.
It can be applied to version 1.34.6 with the following:

 cd <bacula-source>
 patch -p0 <1.34.6-ACL.patch
 make
 make install
 ...

Index: src/filed/job.c
===================================================================
RCS file: /cvsroot/bacula/bacula/src/filed/job.c,v
retrieving revision 1.78
diff -u -r1.78 job.c
--- src/filed/job.c	12 Jun 2004 12:38:16 -0000	1.78
+++ src/filed/job.c	15 Oct 2004 07:20:28 -0000
@@ -833,6 +833,9 @@
       case 'k':
 	 fo->flags |= FO_KEEPATIME;
 	 break;
+      case 'A':
+	 fo->flags |= FO_ACL;
+	 break;
       case 'V':                  /* verify options */
 	 /* Copy Verify Options */
          for (j=0; *p && *p != ':'; p++) {
